### ISSUE TITLE
Incomplete action is silently ignored

### ISSUE THREAD

**Issue by shykes** [dagger/dagger#1572]
## Problem

When I call `docker.#Run` with a missing image, it silently skips the underlying task, instead of returning an error. This caused me to miss a bug in my code, because the tests that should have caught it were silently ignored instead...

NOTE: this is probably not specific to `docker.#Run`. I didn't have time to investigate.

## Solution

Incomplete tasks should cause an error, or at least a very visible warning.

**Comment by aluzzardi** [comment]
I believe this should have been fixed by #2266

> When I call docker.#Run with a missing image, it silently skips the underlying task, instead of returning an error. This caused me to miss a bug in my code, because the tests that should have caught it were silently ignored instead...

By that you mean not setting the `input` field?

If so, that's a test case of #2266 which is actually fixed https://github.com/dagger/dagger/pull/2266#issuecomment-1106817508

```cue
dagger.#Plan & {
	actions: test: docker.#Run
}
```

Will result in:

```console
failed to load plan: "actions.test.input" is not set:
    ./test:13:11
    ./cue.mod/pkg/universe.dagger.io/docker/run.cue:13:2
```

**Comment by helderco** [comment]
Correct, what's described here is covered by the tests in #2266.
